KDIC/BlogUITableExtensionViewController.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  BlogUITableExtensionViewController.swift
+//  KDIC
+//
+//  Created by Shaun Mataire on 4/24/16.
+//  Copyright © 2016 Colin Tremblay. All rights reserved.
+//
+
+import UIKit
+import SafariServices
+
+extension BlogViewController: UITableViewDataSource {
+    
+    func numberOfSectionsInTableView(tableView: UITableView) -> Int {
+        return 1
+    }
+    
+    func tableView(tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        
+        return (self.articles.count)
+    }
+    
+    func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCellWithIdentifier("BlogPostsCell", forIndexPath: indexPath) as! BlogTableCell
+        
+        cell.article = articles[indexPath.row]
```

Comment:
Are these Blogs or Articles? You use a `BlogTableCell` to display `Article`s; pick one and stick to it.